tests/test_utils.py

```diff
@@ -108,6 +108,13 @@ def test_groupable_ordered_dict_copy(god):
     assert god == god2
 
 
+def test_groupable_ordered_dict_deepcopy(god):
+    """Test that a GroupableOrderedDict can be copied deeply."""
+    god2 = copy.deepcopy(god)
+
+    assert god == god2
```

Comment:
you should probably check that `id(god) != id(god2)` and that modification of `god2` doesn't change `god`.